source/Grid/Grid.js

```diff
@@ -63,6 +64,12 @@ export default class Grid extends Component {
     noContentRenderer: PropTypes.func.isRequired,
 
     /**
+     * Optional callback invoked on mouse-wheel activity.
+     * (event: SyntheticEvent): void
+     */
+    onWheel: PropTypes.func,
```

Comment:
Understood. I'm not necessarily against opt-in experimental things like this... it just feels a bit odd. We don't pass-through other event handlers. The only other similar props (`onScroll` and `onSectionRendered`) are used in a totally different way- to notify of changes to the `Grid` and not its inner components.